starters/next-drupal-starter/pages/500.js

```diff
@@ -0,0 +1,13 @@
+import Link from "next/link";
+
+export default function Custom500() {
```

Comment:
I think this is an improvement for now, but what this really feels like is a preview error route rather than a generic 500.

I think I'm going to create an enhancement ticket for this. If this was a preview error route, we could do things like pass params to provide feedback like 'the path does not match' or 'the requested preview path does not exist'

Reply:
Created https://getpantheon.atlassian.net/browse/DB-2874 for this.